packages/ap-sports/src/lib/schedule.ts

```diff
@@ -0,0 +1,51 @@
+import { datetime, errors } from '@linkpool/shared'
+
+import { statusToStatusId } from './constants'
+
+import type { Game, GameCreate, GameResolve } from './types'
+
+export function getGameCreate(game: Game): GameCreate {
+  const { home, away } = game
+  const gameCreate = {
+    gameId: game.id.replace(/-/g, ''),
```

Comment:
As agreed, `getGameCreate` and `getGameResolve` must have `gameId` as `bytes32`:
```typescript
    gameId: uuidToBytes32(game.id),
```